pkg/provider/vault/vault.go

```diff
@@ -558,6 +558,8 @@ func (v *client) readSecret(ctx context.Context, path, version string) (map[stri
 func (v *client) newConfig() (*vault.Config, error) {
 	cfg := vault.DefaultConfig()
 	cfg.Address = v.store.Server
+	// In a Controller runtime context, we rely on the reconciliation process for retrying
+	cfg.MaxRetries = 0
```

Comment:
Here, if we let the default config, in case of issue, we are going to have on each reconcile 3 requests... (2 exponential backoff...)